dnsmasq/Dockerfile.cross

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM BASEIMAGE
+MAINTAINER Girish Kalele <gkalele@google.com>
+ADD dnsmasq.conf /etc/dnsmasq.conf
+ADD src/dnsmasq /usr/bin/dnsmasq
+ADD DNSMASQVERSION.tar.xz /DNSMASQVERSION.tar.xz
```

Comment:
I wanted to include the sources used to build it, don't know the normal practice for building containers - I will remove them.